cmd/livepeer/starter/starter.go

```diff
@@ -1143,6 +1060,179 @@ func StartLivepeer(ctx context.Context, cfg LivepeerConfig) {
 		}()
 	}
 
+	var aiCaps []core.Capability
```

Comment:
@victorges I had to move this over to make the PriceFeed available. In the future, we might want to clean up this script to separate the pricing addition from the model loading. For now, I aimed to minimize conflicts with our existing open commits.

Reply:
Got it. It got a little harder to review but I focused only on the pricing stuff and assumed the rest was copied.

Reply:
Yea the rest was kept the same 👍🏻.